idaes/power_generation/unit_models/heat_exchanger_3streams.py

```diff
@@ -348,21 +348,23 @@ def _make_performance(self):
                                     doc='Temperature difference '
                                     'at side 3 outlet')
 
-        # Driving force side 2
+        # Driving force side 2 (Underwood approximation)
         @self.Constraint(self.flowsheet().config.time,
-                         doc="Log mean temperature difference calculation")
+                         doc="Log mean temperature difference calculation "
+                         "using Underwood approximation")
         def LMTD_side_2(b, t):
             return b.temperature_driving_force_side_2[t] == \
-                ((b.side_2_inlet_dT[t]**0.3241
-                  + b.side_2_outlet_dT[t]**0.3241)/1.99996)**(1/0.3241)
+                ((b.side_2_inlet_dT[t]**0.333333
+                  + b.side_2_outlet_dT[t]**0.333333)/2)**(1/0.333333)
```

Comment:
It probably doesn't matter much, but I would have used `(1/3)` instead of `0.333333`, an especially made the `1/0.333333` just `3`.

Reply:
We should probably insist on 1/3 or 3 here if there is any hope of passing the unit of measure consistency check.

On a side note, we may want to experiment with expressions like this and see what works and what doesn't.  On some of these sorts of things we may need to remove and reattach units.

Reply:
done